src/main/kotlin/no/nav/familie/ef/søknad/søknad/mapper/BarnMapper.kt

```diff
@@ -212,12 +210,6 @@ object BarnMapper : MapperMedVedlegg<List<Barn>, List<Søknadbarn>>(BarnaDine) {
         Søknadsfelt(
             Språktekster.Samvær.hentTekst(),
             Samvær(
-                spørsmålAvtaleOmDeltBosted = forelder.avtaleOmDeltBosted?.tilSøknadsfelt(),
-                avtaleOmDeltBosted =
-                    dokumentfelt(
-                        DELT_BOSTED,
-                        dokumentMap,
-                    ),
```

Comment:
Spørsmålet er om vi skal fjerne mappingen av spørsmålet når vi tar i mot en søknad nå, eller vente med dette til vi er ganske sikre på at det ikke finnes noen mellomlagrede verdier igjen 🤔 

Jeg heller egentlig mot at det er enklest å bare fjerne det med en gang men jeg er åpen for innspill

Reply:
Slik vi har løst det NÅ så er det mest riktig å fjerne det. For det vil ikke fremkomme på oppsummering eller i søknaden dersom du har en mellomlagret søknad og svart på det spørsmålet.